cpp/oneapi/dal/test/engine/dataframe_common.cpp

```diff
@@ -261,6 +263,45 @@ class dataframe_builder_action_fill_uniform : public dataframe_builder_action {
     std::int64_t seed_ = 7777;
 };
 
+class dataframe_builder_action_read_csv : public dataframe_builder_action {
+public:
+    explicit dataframe_builder_action_read_csv(std::string name) : name_(name) {}
+
+    std::string get_opcode() const override {
+        return fmt::format("read_csv({})", name_);
+    }
+
+    dataframe_impl* execute(dataframe_impl* df) const override {
+        if (!df) {
+            throw invalid_argument{ "Action read_csv got null dataframe" };
+        }
+
+        const auto data_table = dal::read<dal::table>(dal::csv::data_source{ name_ });
+
+        if (df->get_row_count() != data_table.get_row_count()) {
+            throw invalid_argument{ "Data_frame row_count should be equal to dataset row_count" };
+        }
+        if (df->get_column_count() != data_table.get_column_count()) {
+            throw invalid_argument{
+                "Data_frame column_count should be equal to dataset column_count"
+            };
+        }
+
+        const float* data_table_ptr =
+            dal::row_accessor<const float>{ data_table }.pull().get_mutable_data();
```

Comment:
What is the reason to call `.get_mutable_data()` if it will be casted into `const float*`?